test/Dialect/FIRRTL/parse-errors.fir

```diff
@@ -749,3 +749,8 @@ circuit UnsupportedStringEncodedIntegerLiterals:
     output foo: UInt<8>
     ; expected-error @below {{String-encoded integer literals are unsupported after FIRRTL 3.0.0}}
     connect foo, UInt("h2a")
+
+// -----
+circuit UnknownTypeAlias:
+  module UnknownTypeAlias:
+    output x : Foo ; expected-error {{type identifier `Foo` is not declared}}
```

Comment:
Another test for `if (!getConstants().aliasMap.insert({id, type}).second)
    return emitError(loc) << "type alias " << name << " is already defined";` would be great.